src/main/java/doore/login/exception/LoginExceptionType.java

```diff
@@ -0,0 +1,27 @@
+package doore.login.exception;
+
+import doore.base.BaseExceptionType;
+import org.springframework.http.HttpStatus;
+
+public enum LoginExceptionType implements BaseExceptionType {
+    NOT_FOUND_GOOGLE_ACCESS_TOKEN_RESPONSE(HttpStatus.NOT_FOUND, "Google Access Token 요청에 대한 응답이 null입니다."),
+    INVALID_ACCESS_TOKEN(HttpStatus.UNAUTHORIZED, "유효하지 않은 Access Token입니다.");
```

Comment:
```suggestion
    NOT_FOUND_GOOGLE_ACCESS_TOKEN_RESPONSE(HttpStatus.NOT_FOUND, "Google Access Token 요청에 대한 응답이 null입니다."),
    INVALID_ACCESS_TOKEN(HttpStatus.UNAUTHORIZED, "유효하지 않은 Access Token입니다."),
    ;
```

정말 사소하지만 이렇게 하는건 어떨까용?.? 나중에 Enum 을 추가할 때 편할 것 같다고 개인적으로 생각합니다~

Reply:
헉 전에 보름님도 같은 의견을 주셨었는데, 전 개인적으로 라인이 `,;`로 끝나는 게 지저분해보여서 이렇게 하는 편입니다🥲
개발자들 중 enum 문법을 모르는 사람은 없으니 ,의 여부가 사용성에 큰 영향을 줄 것이라는 생각은 들지 않네요